### PR TITLE
Openslide testdata

### PR DESCRIPTION
**Pull request by erikogabrielsson**
Change test data for ndpi and svs to publicly available openslide test data. Ignore test if test file not found.

### CODE REVIEW

**Comment by ap--** [comment]
Once this is merged, I can prepare a PR that enables cross platform ci tests: https://github.com/ap--/opentile/tree/ci-tests

Also, quick note:
According to my tests, using https://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/CMU-1.svs as the test image at `./svs/CMU-1/CMU-1.svs` leads to two failing tests. What's the md5sum of your `./svs/CMU-1/CMU-1.svs` @erikogabrielsson

**Comment by ap--** [comment]
the two failing tests are:
```

**Comment by erikogabrielsson** [comment]
Hi,
The test files for the jpeg test were still the old ones. I have now updated the test. Thanks for noticing.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,23 @@ tile = tiler.get_tile(0, (0, 0))
...
+
+## Setup environment for development
+Requires poetry and pytest and pytest-watch installed in the virtual environment.
+
+```console
+$ git clone https://github.com/imi-bigpicture/opentile.git
+$ poetry install
+```
+
+Set the OPENTILE_TESTDIR environment variable to your test data directory. The tests for svs and ndpi uses [openslide testdata](https://openslide.cs.cmu.edu/download/openslide-testdata/). The test data for philips tiff is currently not publically avaiable.
```
Comment (jesper-molin) [inline README.md:66]: Now I have to read the code to know how to organize the slides. Would it be an improvement to:

- Imitate the folder structure from the link
- Add to the readme that the folder structure should be kept

Or shortterm/longterm improvement. Create dev-script that fetches the files you want and puts it in the folder structure you want and then the tests just runs?

**Review by jesper-molin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (erikogabrielsson) [reply README.md]: Added a script for downloading and/or checking openslide testdata.

**Comment by erikogabrielsson** [comment]
Closes #24

**Pull request merged**